ecc/bls12381/ff/cyclo12.go

```diff
@@ -0,0 +1,72 @@
+package ff
+
+import "fmt"
+
+// Cyclo12 represents an element of the 12-th cyclotomic group.
+type Cyclo12 [2]Fp6
+
+func (z Cyclo12) String() string           { return fmt.Sprintf("\n0: %v\n1: %v", z[0], z[1]) }
+func (z *Cyclo12) SetOne()                 { z[0].SetOne(); z[1].SetZero() }
```

Comment:
This is the identity, and I've found it nice to have the generator as well, e.g e(g1, g2).

Reply:
updated now